api/src/utils/task-audit-record.test.js

```diff
@@ -52,11 +52,11 @@ describe('TaskAuditRecord', () => {
 
       expect(mockFindOneAndUpdate).toHaveBeenCalledWith(
         { _id: '456' },
-        { $set: { _id: '456', ...{ param1: 3, param2: 4, read: ROLES.ADMIN_ROLES, write: [ROLES.SYSADMIN] } } },
+        { $set: { _id: '456', ...{ param1: 3, param2: 4, read: utils.ApplicationRoles.ADMIN_ROLES, write: [utils.ApplicationRoles.SYSADMIN] } } },
```

Comment:
Some of these are still using ADMIN_ROLES which was removed.

Reply:
Fixed in 59a3d4f.